packages/mobile/src/store/sagas.ts

```diff
@@ -17,16 +18,19 @@ export default function* rootSaga() {
   const sagas = [
     // config
     ...backendSagas(),
-    // ...accountSagas(),
+    ...accountSagas(),
```

Comment:
Did you notice any worse perf with everything enabled like this?

Reply:
Across the app no. There's a slight hiccup when pressing the sign in button, but i think fix-forward makes sense for it. Can disable if it's a bad call once it's on the staging app